doc/rst/language/spec/interoperability.rst

```diff
@@ -266,10 +266,10 @@ can be described in Chapel using
 
 .. _Referring_to_External_C_Structs:
 
-Referring to External C Structs
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Referring to External C Structs and C Unions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This isn't a show-stopper, but if we make other edits, let's drop the second "C" here, since I think it's implied that the first carries over (as in the sentence just below).

Reply:
Sure, makes sense!